cms/templates/fragments/standalone-page-bootstrap.html

```diff
@@ -0,0 +1,27 @@
+## mako
+
+## Override the default styles_version to use Bootstrap
+<%! main_css = "css/bootstrap/studio-main.css" %>
```

Comment:
These files should diverge substantially when we implement true Bootstrap for Studio. My preference is to keep them independent.